tools/rosetta/lib/internal/service/online.go

```diff
@@ -25,6 +31,15 @@ func NewOnlineNetwork(network *types.NetworkIdentifier, client crgtypes.Client)
 		return OnlineNetwork{}, err
 	}
 
+	// Get genesis hash from ENV. It should be set by an external script since is not possible to get
+	// using tendermint API
+	genesisHash := os.Getenv(genesisHashEnv)
+	if genesisHash == "" {
+		logger.Error(fmt.Sprintf("Genesis hash env '%s' is not properly set!", genesisHashEnv))
```

Comment:
After this log, should this return an error given we expected the genesis hash which is critical?

Reply:
Not having the genesis hash is not critical right now, however we have a fix ready for a next PR that contemplates the proper genesis block height and 'trust height' depending on the sync method used to sync the node.